benchmarks/imagenet/resnet50/knn_eval.py

```diff
@@ -67,6 +67,7 @@ def knn_eval(
     classifier = KNNClassifier(
         model=model,
         num_classes=num_classes,
+        knn_k=20,
```

Comment:
I believe this is unrelated to the PR, could you remove it again?

Reply:
I removed it